tests/ui/type-alias-impl-trait/in-where-clause.stderr

```diff
@@ -16,7 +16,7 @@ LL | / fn foo() -> Bar
 LL | | where
 LL | |     Bar: Send,
    | |______________^
-   = note: ...which requires revealing opaque types in `[Binder { value: TraitPredicate(<Bar as core::marker::Send>, polarity:Positive), bound_vars: [] }]`...
+   = note: ...which requires revealing opaque types in `[Binder { value: (Alias(Opaque, AliasTy { args: [], def_id: DefId(0:7 ~ in_where_clause[cb1b]::Bar::{opaque#0}) }): core::marker::Send), bound_vars: [] }]`...
```

Comment:
This is a bit unfortunate lol. we actually print the `Ty`s in `TraitPredicate`/`TraitRef` using `Debug` now and the debug printing for aliases is not super concise